geometry_msgs/Pose pose
float64[] joints
---
# Success if empty
string error
